.coderabbit.yml

```
@@ -1,35 +1,15 @@
-    # 자동 코드 리뷰를 활성화합니다.
-    # 기본값: true
     enabled : true
 
-    # 드래프트 PR/MR을 리뷰합니다.
-    # 기본값: false
-    drafts : false
-# =============================================================================
-# KNOWLEDGE BASE
-# 지식 기반 설정입니다.
-# =============================================================================
 knowledge_base :
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)